src/services/stringify.ts

```diff
@@ -0,0 +1,9 @@
+import stringify from 'safe-stable-stringify';
+
+const configuredStringify = stringify.configure({
+    deterministic: true,
+});
+
+export const stringifyJSON = (json: any) => {
+    return configuredStringify(json, null, 2);
+};
```

Comment:
New safe stringify. Moved to a new function just to make sure we can control options/settings in a single location